docs/development/blocks/conditionals/overview.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Conditional blocks"
+sidebarTitle: "Overview"
+description: "A conditional block is associated to another block.
+The conditional block will be executed before the parent block,
+and will influence whether or not the parent block gets executed."
+---
+
+<Note>Requires version `0.8.92` or greater.</Note>
+
+Conditional blocks are part of your pipeline but don’t run as individual steps in your pipeline
+like data loader blocks, transformer blocks, etc.
+However, conditional blocks are associated to other blocks within the pipeline
+(e.g. data loaders, transformers, data exporters, etc).
+
+Before those other blocks are executed,
+any associated conditional blocks will be executed. If any conditional block returns False, the
+parent block will not be executed, and its status will be set as `condition_failed`.
+
+---
+
+## Example
+
+1. Your pipeline has the following blocks:
+    1. `load_data_from_api` (data loader)
+    1. `clean_column_names` (transformer)
+    1. `save_data` (data exporter)
+
+1. Then, you add the following conditional block to your pipeline:
+
+    ```python
+    @condition
+    def check_if_data_exists(*args, **kwargs):
+        # some code...
+    ```
+
+1. You associate the above conditional block to the following block:
+
+    - `clean_column_names` (transformer)
+
+1. When you run the pipeline, it’ll execute the `load_data_from_api` (data loader) block first.
+It completes, but the dataframe returned is empty.
+
+1. Next, the pipeline will execute the conditional block and execute the function `check_if_data_exists`.
+Since there is no data in the dataframe, the function returns False.
+
+1. The `clean_column_names` transformer block is not executed, and the status is set as `condition_failed`.
+The `save_data` data exporter block is also not executed, and is set as `condition_failed`.
+
+---
+
+## How to add conditionals to your pipeline
+
+1. Create a new pipeline or open an existing pipeline.
+1. Edit the pipeline.
+1. On the right side of the page, click the <b>Add-ons</b> icon in the navigation.
+    If you don’t see it, try expanding the right area of the page.
+1. Click the button <b>Conditionals<b>.
+1. Click the button <b>+ Conditional block</b>.
+1. In the conditional block’s code, add a function and decorate it with `@condition`.
+
+### Positional arguments
+
+The positional arguments to the conditional block will be the same as the positional
+arguments to the parent block.
+
+<Note>
+    Positional arguments are only supported in Mage versions `>= 0.8.81`.
```

Comment:
What is it for? Do you copy this from callback block doc? If so, I think that's an issue in callback block previously.